PackDialog.gd

```diff
@@ -0,0 +1,65 @@
+extends Control
+
+const executable_path: String = "./xml_converter/build/xml_converter"
+var protobin_data_folder: String
+var split_protobin_data_folder: String
+var user_data_dir: String 
+onready var output_dialog = $WindowDialog
+onready var output_label = $WindowDialog/ScrollContainer/Label
+
+func _ready():
+	var dir = Directory.new()
+	self.user_data_dir = str(OS.get_user_data_dir())
+	self.protobin_data_folder = self.user_data_dir.plus_file("packs")
+	self.split_protobin_data_folder = self.user_data_dir.plus_file("protobins")
+	if not dir.dir_exists(self.protobin_data_folder):
+		var success = dir.make_dir(self.protobin_data_folder)
+		if success != OK:
+			print("Error: Could not create data folder:", self.protobin_data_folder)
+
+
+func clear_directory_contents(dir_path: String) -> void:
```

Comment:
This is **BY FAR** the most dangerous function you have ever written. Please add some sanity checks to it to make sure we are not recursively deleting the wrong directory/whole computer

Reply:
I realize that and this should have not been kept in. I will delete.